build.sbt

```diff
@@ -19,13 +17,8 @@ lazy val microservice = Project(appName, file("."))
       compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
       "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
     ),
-    PlayKeys.playDefaultPort := 8421,
-    ScoverageKeys.coverageMinimumStmtTotal := 30,
```

Comment:
why is the coverage being removed? same comment as the wrapper-data PR

Reply:
CodeCoverageSettings file provide all the required parameters.